mahta/JS/JS DOM-manipulation/script.js

```diff
@@ -0,0 +1,116 @@
+function generateRandomColor(y, b) {
+  const yellow = Math.floor(Math.random() * (y - 155) + 155);
+  const blue = Math.floor(Math.random() * (b - 100) + 100);
+  return `rgb(${yellow}, ${yellow}, ${blue})`;
+}
+
+function onButtonClickSet() {
+  document.body.innerHTML = "";
+  const buttonTag = document.createElement("button");
+  buttonTag.textContent = "Reset";
+  buttonTag.addEventListener("click", firstPage);
+  document.body.appendChild(buttonTag);
```

Comment:
When you click the button, you should use DOM to change color or delete without resetting the page and creating new elements!